File: Python/0224. 连续字母.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            n = input()
            if n.isdigit() == False:
                break
            n = int(n)
            for _ in range(n):
                ss = input()
                ss += '-'
                result = 0
                temp = 0
                last = '0'
                for c in ss:
                    if c == last:
                        temp += 1
                    else:
                        result = max(result, temp)
                        temp = 1 
                        last = c
                print(result)
        except EOFError:
            break
    
if __name__ == "__main__":
    func()

File: C++/0224. 连续字母.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    int n;
    cin >> n;
    for(int i=0;i<n;++i) {
        string ss;
        cin >> ss;
        ss += '-';
        int result = 0;
        int temp = 0;
        char last = '0';
        for(auto c:ss) {
            if(c == last) {
                temp += 1;
            } else {
                result = max(result, temp);
                temp = 1;
                last = c;
            }
        }
        cout<<result<<endl;
    }
    
    return 0;
}
